configure.ac

```diff
@@ -470,7 +470,7 @@
         LDFLAGS="$LDFLAGS -lgxpci -lgxio -ltmc"
         ],
         [AC_MSG_RESULT([no])])
-
+  
```

Comment:
is this necessary?

Reply:
Nope, unintended whitespace from when we were attempting to add inotify support. Missed when that was removed.